modules/fedimint-wallet-client/src/lib.rs

```diff
@@ -162,24 +183,36 @@ impl ClientModuleInit for WalletClientInit {
     }
 
     async fn init(&self, args: &ClientModuleInitArgs<Self>) -> anyhow::Result<Self::Module> {
+        let data = WalletClientModuleData {
+            cfg: args.cfg().clone(),
+            module_root_secret: args.module_root_secret().clone(),
+        };
+
         let rpc_config = self
             .0
             .clone()
             .unwrap_or(WalletClientModule::get_rpc_config(args.cfg()));
 
-        // FIXME: reactivate key derivation once we implement recovery
```

Comment:
While the recovery is not implemented yet, re-creating same deposit addresses should work perfectly fine now, and we will just detect new deposits.